src/SdnDiagnostics.psm1

```diff
@@ -953,209 +958,140 @@ function Start-SdnDataCollection {
+                    $netDiagLogShareParams = @{
+                        LogDir           = $logDir
+                        OutputDirectory  = $outputDir
+                        FromDate         = $FromDate
+                        ToDate           = $ToDate
+                        FolderNameFilter = $filterArray
+                    }
 
-                # create an array of folders to collect the logs from leveraging the common configuration
-                $logDir = @()
-                $commonConfig.DefaultLogFolders | ForEach-Object {
-                    $logDir += Join-Path -Path $diagLogNetShare -ChildPath $_
+                    Get-SdnDiagnosticLogFile @netDiagLogShareParams
                 }
-                $ncNodeFolders | ForEach-Object {
-                    $ncNetBiosName = (Get-ComputerNameFQDNandNetBIOS -ComputerName $_).ComputerNameNetBIOS
-                    $logDir += Join-Path -Path $diagLogNetShare -ChildPath $ncNetBiosName
-                }
-                $logDir = $logDir | Sort-Object -Unique
-
-                # create parameters for the Get-SdnDiagnosticLogFile function
-                $netDiagLogShareParams = @{
-                    LogDir           = $logDir
-                    OutputDirectory  = $outputDir
-                    FromDate         = $FromDate
-                    ToDate           = $ToDate
-                    FolderNameFilter = $filterArray
-                }
-
-                Get-SdnDiagnosticLogFile @netDiagLogShareParams
             }
         }
 
-        if ($IncludeNetView) {
-            "Collect Get-NetView logs for {0}" -f ($filteredDataCollectionNodes -join ', ') | Trace-Output
-            $splat = @{
-                ComputerName = $filteredDataCollectionNodes
-                Credential   = $Credential
-                ScriptBlock  = $collectNetViewSB
-                ArgumentList = @($tempDirectory.FullName)
-                AsJob        = $true
-                PassThru     = $true
-                Activity     = 'Invoke Get-NetView'
-            }
-            $null = Invoke-PSRemoteCommand @splat
-        }
-
-        foreach ($node in $filteredDataCollectionNodes) {
+        foreach ($node in $dataCollectionNodes.Name) {
             [System.IO.FileInfo]$formattedDirectoryName = Join-Path -Path $OutputDirectory.FullName -ChildPath $node.ToLower()
             Copy-FileFromRemoteComputer -Path $tempDirectory.FullName -Destination $formattedDirectoryName.FullName -ComputerName $node -Credential $Credential -Recurse -Force
-            Copy-FileFromRemoteComputer -Path "$(Get-WorkingDirectory)\SdnDiagnostics_TraceOutput*.csv" -Destination $formattedDirectoryName.FullName -ComputerName $node -Credential $Credential -Force
+            Copy-FileFromRemoteComputer -Path "$(Get-WorkingDirectory)\SdnDiagTrace*.csv" -Destination $formattedDirectoryName.FullName -ComputerName $node -Credential $Credential -Force
```

Comment:
>SdnDiagTrace [](http://example.com/codeflow?start=71&length=12)

nit: consider not changing names of files , because so many people are looking for files, change of names throws them off. It could even break filters / kusto queries someone might have written.